### PR TITLE
Fix episode navigation, Hide keyboard on drag in search result page

### PR DESCRIPTION
**Pull request by eliezerben**
Fix episode navigation,
On search result page, hide the keyboard when user starts to scroll through the results.

### CODE REVIEW

File: lib/router/router.dart [hunk 1]
```diff
@@ -31,6 +37,11 @@ import '../screens/search/explore_category_page.dart';
     MaterialRoute<void>(page: ContactSupport, path: 'contact-support'),
     MaterialRoute<void>(page: AboutScreen, path: 'about'),
     MaterialRoute<void>(page: FAQ, path: 'faq'),
+    CustomRoute<void>(
```
Comment (andreasgan) [inline lib/router/router.dart:40]: does this need to be here?
If yes, use the variable you created.
If no, you can remove it :)

File: lib/env/env.dart [hunk 2]
```diff
@@ -1,4 +1,6 @@
-part '.env.dart';
+// import 'package:envify/envify.dart';
+
+// part '.env.dart';
```
Comment (andreasgan) [inline lib/env/env.dart:3]: wrong approach. see teams

**Review by andreasgan — CHANGES_REQUESTED** [review]

File: lib/router/router.dart (hunk 1 above)
Comment (eliezerben) [reply lib/router/router.dart]: It's not needed. Forgot to remove it.

File: lib/env/env.dart (hunk 2 above)
Comment (eliezerben) [reply lib/env/env.dart]: Fixed.

**Review by andreasgan — APPROVED** [review]

**Pull request merged**